build/download7z.js

```diff
@@ -0,0 +1,87 @@
+const path = require("path");
```

Comment:
このコードが何をするのかがわかるようなドキュメントを一番上に書いておくと迷子にならないかもと思いました！

```suggestion
/**
 * 7-Zipとそのライセンス情報をダウンロードする。
 */
const path = require("path");
```